automation/e2e-tekton/example-pipelines-test.sh

```diff
@@ -95,8 +95,12 @@ oc wait -n kubevirt --for=condition=Ready pod -l app=http-server --timeout=10m
 echo "Deploy SSP and create sample"
 make deploy
 
-# Deploy tekton task sample
+oc wait -n kubevirt deployment ssp-operator --for condition=Available --timeout 10m
```

Comment:
Can you please remove empty line above that, and rename `echo "Deploy SSP and create sample"` to `echo "Deploy SSP operator"`?